jogo42-cb/jogador.c

```diff
@@ -75,6 +75,10 @@ void MoverJog(GameState* gs)
     {
         gs->jog.pos = posFutura;
     }
+    // Ataque
+    gs->jog.hitbox = gs->jog.pos;
+    gs->jog.hitbox.x = gs->jog.hitbox.x + 60*cos(gs->jog.rot/60);
```

Comment:
60 graus não é igual a 1 radiano. O valor certo é aproximadamente `57.2958`.

Mas também não é pra dividir por esse valor. Em `raymath.h` está definida a constante `DEG2RAD`, que tem o valor tal que quando multiplicado por um ângulo em graus, o resultado é esse mesmo ângulo em radianos (esse valor é `PI / 180.0f`).

Obs: Existe também a constante `RAD2DEG`, para fazer a conversão inversa. O `2` nesses nomes representam a palavra inglesa "to".

Trecho relevante de `raymath.h`:
```c
/* ... */

//----------------------------------------------------------------------------------
// Some basic Defines
//----------------------------------------------------------------------------------
#ifndef PI
    #define PI 3.14159265358979323846f
#endif

#define DEG2RAD (PI/180.0f)
#define RAD2DEG (180.0f/PI)

/* ... */
```

```suggestion
    gs->jog.hitbox.x = gs->jog.hitbox.x + RAD2DEG * cos(gs->jog.rot * DEG2RAD);
```

Reply:
Consertei com commit 9523dee